### DISCUSSION TITLE
ベースアイテム(k_infoのオブジェクト群)調整案について (3.0.0α→3.0.0β)

### DISCUSSION THREAD

**Discussion by iks3** [hengband/hengband#300] [Ideas]
ベースアイテムの階層、レア度、能力の調整について。

**Comment by iks3** [comment]
### アミュレット編
|名称|変更点|対応|
|---|---|---|
|知性|レア度50/4→30/4|α11|
|探索|レベル15、レア度30/4→15/8:30/4、+探赤(MAX8)、弱ESP追加|α11|
|魔法道具支配|レア度35/4→20/4:35/4|α11|
|反魔法|レア度40/4→30/4:50/4|α11|
|耐性|レア度50/4→35/4:50/4|α11|
|賢者|レア度50/4:80/3→30/4:60/3|α11|
|反射|レア度60/4→40/3|α11|
|テレパシー|レア度60/4→50/3|α11|

### 指輪編
|名称|変更点|改善案|対応|
|---|---|---|---|
|探索|+探赤隠(MAX4)|||
|魅力維持|名称:魅力、+魅(MAX+4)、維:なし|||
|知力維持|名称:知力、+知(MAX+4)、維:なし|||
|賢さ維持|名称:賢さ、+賢(MAX+4)、維:なし|||
|腕力維持|廃止(レア度0)または差し替え|||
|器用さ維持|廃止(レア度0)または差し替え|||
|耐久力維持|廃止(レア度0)または差し替え|||
|耐盲目|レア度60/2→30/2、価格7500→4000||α11|
|能力維持|レア度60/5→35/4、耐時間逆転、経験値維持||α11|
|耐劣化|レア度90/10→40/4、価格15000→7500||α11|
|耐毒|レア度40/2→30/4:40/4||α11|
|消費魔力減少|レア度80/8→65/6||α11|
|王者の加護|レア度100/8→75/6||α11|

**Reply by daradarach** [reply]
・腕/器/耐の指輪は+6まであったかと思いますが知/賢/魅の指輪がMAX+4なのはアミュレットもあることからのバランス調整ですか？
・廃止見込みの単体能力維持群は各能力値上昇指輪専用で○○維持の～のエゴなんてのはいかがでしょうか
・能力維持の指輪も単体装備はあまり考えないと思います。+腕知賢器耐魅(MAX+1)とかどんなもんでしょうか。フラキアが立つ瀬なくなる？

**Reply by iks3** [reply]
> ・腕/器/耐の指輪は+6まであったかと思いますが知/賢/魅の指輪がMAX+4なのはアミュレットもあることからのバランス調整ですか？

そんな感じです。

**Reply by gold-onion** [reply]
単体能力の奴は維持も持たせてもいいのでは
能力維持の指輪は生命力維持と耐時間逆転を標準で持つようにするのはどうでしょうか

**Reply by iks3** [reply]
> 能力維持の指輪は生命力維持と耐時間逆転を標準で持つよう

なるほど。実現の際はエゴの方をどうにかするを忘れないようにしないといけませんね。

**Comment by iks3** [comment]
### ロッド編
|名称|変更点|
|---|---|
|周辺感知|レベル65→35、レア度33/16:65/4→30:16/55/4|
|癒し|レベル65→35、レア度33/32:65/8→30/16:50/8|
|体力回復|レベル80→50、レア度40/32:80/8→40/24:80/6|
|全復活|レベル80→50、レア度40/64:80/16→40/21:65/12|
|スピード|レベル60→45、レア度48/64:95/16→40/32:75/12|

**Reply by Aquashrimp** [reply]
全感知ももう少し出やすくするのはどうでしょう？

**Comment by iks3** [comment]
### 武器編
|名称|変更点|
|---|---|
|クォピス|名称をケペシュまたはコペシュにする|

**Comment by gold-onion** [comment]
今はドラゴン・スケイルメイルはエゴが出ず☆の出現率も他の鎧よりも低くなっています
そのためバランスとかの一部を除いて耐性が貧弱で能力上昇も無く、現状生成されたらほぼハズレになってます
鎧以外のドラゴン装備と同様に☆の出現率を上げたりエゴ解禁できないでしょうか
全耐性系のエゴは除いた方がいいかもしれません

**Reply by deskull-m** [reply]
アリとは思いますね。対応は大分先になるとは思いますがIssue化しておきます。

**Comment by daradarach** [comment]
新装備フラグ:魔法防御上昇 #198 実装の暁にはCoMF(魔法を防ぐクローク)を！

**Reply by Hourier** [reply]
実装済です (担当者が別途いるため調整度合いは不明瞭)

**Comment by deskull-m** [comment]
OSDNのチケットに　https://osdn.net/projects/hengband/ticket/41346 というものがありましたが、このDisscussionと重複している案件と武蔵セットの要望が既に解決しつつある＆別件で済ませているようなので重複扱いとして完了します。

**Comment by Aquashrimp** [comment]
魔法を防ぐクロークですが、自動拾いにおける「レアなアイテム」に含めても良いと思います

**Reply by habu1010** [reply]
備忘： 以下で実装済み

- #1271

**Comment by oguchi2918** [comment]
透明物体感知の薬ですが、出現階層1Fか店売りを希望します。
ついでに透明物体感知の巻物・杖は廃止してよいかと思います。

理由は、
・1ターンだけ見えてもあまり役に立たない・ストレス
・透明体視認を持たない種族が選択しやすくなる
・序盤装備するエゴ武器の選択幅が増える
・イーク1Fで必要なら集める枠

**Reply by Hourier** [reply]
> 1ターンだけ見えてもあまり役に立たない

まさにここはおっしゃる通りです
今後の検討事項にします (杖も薬と同じように12+d12ターン持たすようにする等)

**Reply by Hourier** [reply]
杖と薬を同一の効果ターンにしました
巻物はこれといったアイディアがないので保留します

なお、一般にアイテム/モンスター/アーティファクトの廃止はよほどの例外事項がない限り却下します
理由はシステム上欠番を作るのが非常に危険であるためです (旧バージョンからのセーブデータコンバート等)

**Comment by oguchi2918** [comment]
調査のロッド(等)で表示される情報ですが、
非ユニークモンスターに抹殺耐性が付いていた場合、表示を希望します。

1/13で耐性が付いた時点でメッセージが出ないのは昔からの仕様なので、
意図的に隠しているのかもしれませんが調査したら情報は明かして欲しい。

**Comment by gitgita6** [comment]
レアでありながらイマイチ有り難みが薄いベースアイテムに基礎能力を付加する案です。
なまじレアなのでエゴや☆も出会いにくいので実用されにくいのを救済で

・天上のクローク　浮遊
（軽さがウリなのはわかるが、クローク枠全体が軽いのでありがたみが薄い）

・アダマンタイト鎧　耐久＆腕力維持　あるいはそのどっちか

・ミスリル鎧　+道　（魔力支配の小手みたく）
　あるいは麻痺知らず
　あるいは透明視
　こっちはアダマンタイトとの差別化で魔法っぽい能力を想定

**Reply by habu1010** [reply]
良さそうなので Issue 化しておきました。
- #1807

**Comment by Aquashrimp** [comment]
籠手枠に腕輪を追加するのはどうでしょうか
器用補正などがなくても魔法を阻害することはなく、エゴは籠手系とは別にする感じで
